app.go

```diff
@@ -390,6 +390,15 @@ type Config struct {
 	//
 	// Optional. Default: DefaultMethods
 	RequestMethods []string
+
+	// EnableSplittingOnParsers splits the query/body/header parameters by comma when it's true.
+	// For example, you can use it to parse multiple values from a query parameter like this:
+	//   /api?foo=bar,baz == foo[]=bar&foo[]=baz
+	// If there is an escape character before the comma, parsers won't split it like the example below:
```

Comment:
Do we need to support escape chars? It has minor performance penalty (1 alloc)

Reply:
no we do not have to -> no escape support

Reply:
but maybe we should already now use a struct for parser configurations for possible later configurations ?

ParserOptions.EnableSplitting

Reply:
> but maybe we should already now use a struct for parser configurations for possible later configurations ?
> 
> ParserOptions.EnableSplitting

It inludes config fields related to schema decoder. Should we do it?

Reply:
Sounds good